python/ThirteenTeV/DisplacedSUSY/DisplacedSUSY_stopToLD_M_XXX_YYYmm_TuneCP5_13TeV_pythia8_cff.py

```diff
@@ -0,0 +1,857 @@
+FLAVOR = 'stop'
+COM_ENERGY = 13000. # GeV
+MASS_POINT = XXX   # GeV
+CROSS_SECTION = ZZZ # pb
+CTAU_POINT = YYY # mm
+SLHA_TABLE = """## Important note!
+## This file has been modified by hand to give the gluino and the
+## stop_1 a very narrow width, such that it can be used to try out
+## the R-hadron machinery. It is not a realistic SUSY scenario.
+##
+##******************************************************************
+##                      MadGraph/MadEvent                          *
+##******************************************************************
+##                                                                 *
+##  param_card corresponding the SPS point 1a (by SoftSusy 2.0.5)  *
+##                                                                 *
+##******************************************************************
+## Les Houches friendly file for the (MS)SM parameters of MadGraph *
+##      SM parameter set and decay widths produced by MSSMCalc     *
+##******************************************************************
+##*Please note the following IMPORTANT issues:                     *
+##                                                                 *
+##0. REFRAIN from editing this file by hand! Some of the parame-   *
+##   ters are not independent. Always use a calculator.            *
+##                                                                 *
+##1. alpha_S(MZ) has been used in the calculation of the parameters*
+##   This value is KEPT by madgraph when no pdf are used lpp(i)=0, *
+##   but, for consistency, it will be reset by madgraph to the     *
+##   value expected IF the pdfs for collisions with hadrons are    *
+##   used.                                                         *
+##                                                                 *
+##2. Values of the charm and bottom kinematic (pole) masses are    *
+##   those used in the matrix elements and phase space UNLESS they *
+##   are set to ZERO from the start in the model (particles.dat)   *
+##   This happens, for example,  when using 5-flavor QCD where     *
+##   charm and bottom are treated as partons in the initial state  *
+##   and a zero mass might be hardwired in the model definition.   *
+##                                                                 *
+##       The SUSY decays have calculated using SDECAY 1.1a         *
+##                                                                 *
+##******************************************************************
+#
+BLOCK DCINFO  # Decay Program information
+     1   SDECAY      # decay calculator
+     2   1.1a        # version number
+#
```

Comment:
please delete such lines

Reply:
sorry, can you tell me exactly which lines? it's not clear to me from github. L43-46 inclusive? thanks!